section4/exercises/burrito.rb

```diff
@@ -11,9 +11,21 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+  def add_topping
+    "lime"
```

Comment:
These methods do not change the values of any properties. You will need assignments to do that. You can also use parameters so that these properties aren't changed to the same thing every time these methods are called.